cli/src/pcluster/resources/imagebuilder/update_and_reboot.yaml

```diff
@@ -322,13 +327,6 @@ phases:
                  echo "SSM agent is installed by default"
               fi
 
-      - name: RebootStep
```

Comment:
There are other packages we install after the previous reboot step, so I dont think we should remove the last reboot step.

Reply:
The two steps should have different names